### PR TITLE
docs: improving developer experience

### PR DESCRIPTION
**Pull request by a0m0rajab**
## What does this PR do?

This PR helps to improve the developer experience by adding the next: 
- Feedback if conventional commit was not followed 
- Welcoming new users

relates #9343 

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)

## How should this be tested?

This can be tested when a new PR is opened or a new Issue has been opened by first contributors. 

## Mandatory Tasks
- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: .github/workflows/semantic-pull-requests.yml [hunk 1]
```diff
@@ -9,13 +9,39 @@ on:
...
+        id: lint_pr_title
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: marocchino/sticky-pull-request-comment@v2
+        # When the previous steps fails, the workflow would stop. By adding this
+        # condition you can continue the execution with the populated error message.
+        if: always() && (steps.lint_pr_title.outputs.error_message != null)
```
Comment (keithwillcode) [inline .github/workflows/semantic-pull-requests.yml:27]: Do we actually need this `always()`?

File: .github/workflows/semantic-pull-requests.yml [hunk 2]
```diff
@@ -9,13 +9,39 @@ on:
...
+            ${{ steps.lint_pr_title.outputs.error_message }}
+            ```
+
+      # Delete a previous comment when the issue has been resolved
+      - if: ${{ steps.lint_pr_title.outputs.error_message == null }}
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:   
+          header: pr-title-lint-error
+          message: |
+            Thanks you for following the naming! 🙏
```
Comment (keithwillcode) [inline .github/workflows/semantic-pull-requests.yml:47]: Change "Thanks you" to "Thank you".
Add "conventions"

Should read as "Thank you for following our naming conventions! 🙏 "

File: .github/workflows/welcome-newers.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: "Welcome new contributors"
```
Comment (keithwillcode) [inline .github/workflows/welcome-newers.yml:1]: Let's change this filename to be `welcome-new-contributors`. `welcome-newers` sounds odd

File: .github/workflows/welcome-newers.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ pull_request:
+   types: opened
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+ welcome-message:
+      name: Welcoming New Users
```
Comment (keithwillcode) [inline .github/workflows/welcome-newers.yml:15]: Spacing here is 4 spaces instead of 2

File: .github/workflows/welcome-newers.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      runs-on: ubuntu-latest
+      timeout-minutes: 10
+      needs: semantics
+      if: github.event.action == 'opened'
+      steps:
+        - uses: actions/first-interaction@v1
+          with:
+            repo-token: ${{ secrets.GITHUB_TOKEN }}
+            pr-message: |-
+              Thank you for making your first Pull Request and taking the time to improve cal.com ! ❤️🎉
```
Comment (keithwillcode) [inline .github/workflows/welcome-newers.yml:25]: cal.com should read as Cal.com

**Review by keithwillcode — CHANGES_REQUESTED** [review]
Thanks a lot for this! Just have a few things we should update.

File: .github/workflows/semantic-pull-requests.yml (hunk 1 above)
Comment (a0m0rajab) [reply .github/workflows/semantic-pull-requests.yml:27]: They used it in the action [example](https://github.com/amannn/action-semantic-pull-request#outputs), I think it's been used to handle[ all cases ](https://docs.github.com/en/actions/learn-github-actions/expressions#always)even if it's failed/canceled.

File: .github/workflows/semantic-pull-requests.yml (hunk 2 above)
Comment (a0m0rajab) [reply .github/workflows/semantic-pull-requests.yml:47]: Done

File: .github/workflows/semantic-pull-requests.yml (hunk 1 above)
Comment (keithwillcode) [reply .github/workflows/semantic-pull-requests.yml:27]: Interesting. Ok!

**Comment by a0m0rajab** [comment]
Updated the PR based on @keithwillcode comments.

**Comment by keithwillcode** [comment]
@a0m0rajab Thanks for the updates! I'm going to test this out.

**Review by keithwillcode — APPROVED** [review]

**Comment by a0m0rajab** [comment]
@keithwillcode Thank you for merging this! Will be working on the other suggestions when I got the bandwidth.

**Pull request merged**